apps/content_filters/filter_condition/filter_condition_parameters.py

```diff
@@ -26,85 +26,99 @@ class FilterConditionParametersResource(Resource):
+                   'operators': ['in', 'nin', 'eq', 'ne', 'lt', 'lte', 'gt', 'gte'],
+                   'values': values['priority'],
+                   'value_field': 'qcode'
+                   },
+                  {'field': 'keywords',
+                   'operators': ['in', 'nin']
+                   },
+                  {'field': 'slugline',
+                   'operators': ['in', 'nin', 'eq', 'ne', 'like', 'notlike', 'startswith', 'endswith']
+                   },
+                  {'field': 'type',
+                   'operators': ['in', 'nin', 'eq', 'ne'],
+                   'values': values['type'],
+                   'value_field': 'qcode'
+                   },
+                  {'field': 'source',
+                   'operators': ['in', 'nin', 'eq', 'ne', 'like', 'notlike', 'startswith', 'endswith']
+                   },
+                  {'field': 'headline',
+                   'operators': ['in', 'nin', 'eq', 'ne', 'eq',
+                                 'ne', 'like', 'notlike', 'startswith', 'endswith']
+                   },
+                  {'field': 'ednote',
+                   'operators': ['in', 'nin', 'eq', 'ne', 'like', 'notlike', 'startswith', 'endswith']
+                   },
+                  {'field': 'body_html',
+                   'operators': ['in', 'nin', 'like', 'notlike', 'startswith', 'endswith']
+                   },
+                  {'field': 'desk',
+                   'operators': ['in', 'nin', 'eq', 'ne'],
+                   'values': values['desk'],
+                   'value_field': '_id'
+                   },
+                  {'field': 'stage',
+                   'operators': ['in', 'nin', 'eq', 'ne'],
+                   'values': values['stage'],
+                   'value_field': '_id'
+                   },
+                  {'field': 'sms',
+                   'operators': ['in', 'nin', 'eq', 'ne'],
+                   'values': values['sms'],
+                   'value_field': 'name'
+                   },
+                  {'field': 'place',
+                   'operators': ['match'],
+                   'values': values['place'],
+                   'value_field': 'qcode'
+                   },
+                  {'field': 'ingest_provider',
+                   'operators': ['eq', 'ne'],
+                   'values': values['ingest_provider'],
+                   'value_field': '_id'
+                   },
+                  {'field': 'embargo',
+                   'operators': ['eq', 'ne'],
+                   'values': values['embargo'],
+                   'value_field': 'name'
+                   }]
+        fields.extend(self._get_vocabulary_fields(values))
+        return ListCursor(fields)
+
+    def _get_vocabulary_fields(self, values):
+        for vocabulary in get_resource_service('vocabularies').get(req=None, lookup=None):
+            if vocabulary[config.ID_FIELD] not in values and vocabulary['type'] == 'manageable':
```

Comment:
imo there should be some filtering, otherwise this gets also crop sizesa, rights info etc

Reply:
Added a filter, please review